tasks/lint_rules/src/eslint-rules.mjs

```diff
@@ -10,62 +10,76 @@ const { Linter } = require('eslint');
+// https://github.com/vercel/next.js/blob/canary/packages/eslint-plugin-next/src/index.ts
+import pluginNext from '@next/eslint-plugin-next';
+// https://github.com/eslint-community/eslint-plugin-promise/blob/v7.1.0/index.js
+import pluginPromise from 'eslint-plugin-promise';
+// https://github.com/veritem/eslint-plugin-vitest/blob/v1.1.9/src/index.ts
+import pluginVitest from 'eslint-plugin-vitest';
+
+const { rules: pluginNAllRules } = pluginNAll;
+
 const {
-  rules: pluginUnicornAllRules,
   configs: pluginUnicornConfigs,
-} = require('eslint-plugin-unicorn');
-// https://github.com/gajus/eslint-plugin-jsdoc/blob/v50.5.0/src/index.js
+  rules: pluginUnicornAllRules,
+} = pluginUnicorn;
+
 const {
-  // @ts-expect-error: Module has no exported member
   rules: pluginJSDocAllRules,
-  // @ts-expect-error: Module has no exported member
   configs: pluginJSDocConfigs,
-} = require('eslint-plugin-jsdoc');
-// https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/src/index.js
-const {
-  rules: pluginImportAllRules,
-  configs: pluginImportConfigs,
-} = require('eslint-plugin-import');
-// https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/v6.9.0/src/index.js
+} = pluginJSDoc;
+
 const {
   rules: pluginJSXA11yAllRules,
   configs: pluginJSXA11yConfigs,
-} = require('eslint-plugin-jsx-a11y');
-// https://github.com/jest-community/eslint-plugin-jest/blob/v28.9.0/src/index.ts
+} = pluginJSXA11y;
+
 const {
   rules: pluginJestAllRules,
   configs: pluginJestConfigs,
-} = require('eslint-plugin-jest');
-// https://github.com/jsx-eslint/eslint-plugin-react/blob/v7.37.2/index.js
-const { rules: pluginReactAllRules } = require('eslint-plugin-react');
-// https://github.com/facebook/react/blob/v18.3.1/packages/eslint-plugin-react-hooks/src/index.js
+} = pluginJest;
+
 const {
-  rules: pluginReactHooksAllRules,
-} = require('eslint-plugin-react-hooks');
-// https://github.com/cvazac/eslint-plugin-react-perf/blob/9bfa930661a23218f5460ebd39d35d76ccdb5724/index.js
+  // @ts-expect-error: Module has no exported member
+  configs: pluginPromiseConfigs,
+  rules: pluginPromiseRules,
+} = pluginPromise;
+
+const { rules: pluginReactAllRules } = pluginReact;
+
+const { rules: pluginNextAllRules } = pluginNext;
+
 const {
   rules: pluginReactPerfAllRules,
   configs: pluginReactPerfConfigs,
-} = require('eslint-plugin-react-perf');
-// https://github.com/vercel/next.js/blob/canary/packages/eslint-plugin-next/src/index.ts
-const { rules: pluginNextAllRules } = require('@next/eslint-plugin-next');
-// https://github.com/eslint-community/eslint-plugin-promise/blob/v7.1.0/index.js
-const {
-  rules: pluginPromiseRules,
-  configs: pluginPromiseConfigs,
-} = require('eslint-plugin-promise');
-// https://github.com/veritem/eslint-plugin-vitest/blob/v1.1.9/src/index.ts
+} = pluginReactPerf;
+
 const {
-  rules: pluginVitestRules,
   configs: pluginVitestConfigs,
-} = require('eslint-plugin-vitest');
+  rules: pluginVitestRules,
+} = pluginVitest;
```

Comment:
Because some of them only use a default export, it's not possible to standardize the way of writing? 👀 

It would be nice to have a comment for that at the beginning, and I think it's fine to remove extra blank lines between them.